bitcoin/testBwm.c

```diff
@@ -1399,7 +1410,7 @@ BRRunTestWalletManagerFileService (const char *storagePath) {
     BRTransactionFree(tx);
     BRTransactionFree(tx2);
     BRSetFree (transactionSet);
-    
+
```

Comment:
Me again?

Reply:
I think so (not a big deal). Do you exclusively edit in XCode? Wouldn't be surprised if the XCode 11 update was somehow to blame.